### PR TITLE
Fix link to drag and drop designer help

### PR DESCRIPTION
**Pull request by m-kuhn**
The help button in "add group or container" was not linking to the correct page

Update:
This now fixes links to all help pages in the vector layer properties and fixes the one pre-existing help link in there to be translation safe.

### CODE REVIEW

File: src/app/qgsvectorlayerproperties.cpp [hunk 1]
```diff
@@ -471,6 +471,26 @@ QgsVectorLayerProperties::QgsVectorLayerProperties(
...
+  mOptsPage_Labels->setProperty( "helpPage", QStringLiteral( "working_with_vector/vector_properties.html#labels-properties" ) );
+  mOptsPage_Diagrams->setProperty( "helpPage", QStringLiteral( "working_with_vector/vector_properties.html#diagrams-properties" ) );
+  mOptsPage_SourceFields->setProperty( "helpPage", QStringLiteral( "working_with_vector/vector_properties.html#source-fields-properties" ) );
+  mOptsPage_AttributesForm->setProperty( "helpPage", QStringLiteral( "working_with_vector/vector_properties.html#attributes-form-properties" ) );
+  mOptsPage_Joins->setProperty( "helpPage", QStringLiteral( "working_with_vector/vector_properties.html#joins-properties" ) );
+  mOptsPage_AuxiliaryStorage->setProperty( "helpPage", QStringLiteral( "working_with_vector/vector_properties.html#auxiliary-storage-properties" ) );
+  mOptsPage_Actions->setProperty( "helpPage", QStringLiteral( "working_with_vector/vector_properties.html#actions-properties" ) );
+  mOptsPage_Display->setProperty( "helpPage", QStringLiteral( "working_with_vector/vector_properties.html#disply-properties" ) );
```
Comment (DelazJ) [inline src/app/qgsvectorlayerproperties.cpp]: typo?
```suggestion
  mOptsPage_Display->setProperty( "helpPage", QStringLiteral( "working_with_vector/vector_properties.html#display-properties" ) );
```

**Review by DelazJ — COMMENTED** [review]
Unfortunately, i'm afraid this is not the only place where links no longer work.
I continue to believe that a tool that checks whether links are still active in docs (with correspondences in both repo) and allows to find broken ones is far better than occasional manual check, but this is a fight I abandon.

**Comment by m-kuhn** [comment]
A script that checks for this sounds like a good thing, I don't think anyone will argue against it.
Checking for the page should be easy.
Checking for the anchor will be a bit harder.

Another question is if this should be on travis. There's a risk that it triggers build failures on unrelated pull requests when things break in a different repo or due to network problems, so I would rather prefer to have an auto-updated table visible somewhere.

And then the final question is how to deal with broken links (correct them on QGIS side or forward via mod rewrite or other means on server side).

**Comment by DelazJ** [comment]
@m-kuhn This is a discussion we [had a year ago](https://github.com/qgis/QGIS-Documentation/pull/2258#issuecomment-352246124) with you already proposing a solution (I'm not skilled to discuss the internals) and for reference there's a starting list at https://github.com/qgis/QGIS-Documentation/issues/1926#issuecomment-416554479

**Comment by m-kuhn** [comment]
We'll need someone with some scripting experience.
Meanwhile, any objections to merging this PR?

**Comment by DelazJ** [comment]
>  Meanwhile, any objections to merging this PR?

None. Thanks for the fix.

**Comment by m-kuhn** [comment]
I guess this justifies a backport

**Comment by DelazJ** [comment]
it should.

**Pull request merged**